<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocketDebugger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_tcpServer">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>TCP Sever</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,3,2" columnstretch="1,4">
        <item row="0" column="0" rowspan="2">
         <widget class="QGroupBox" name="groupBox_bind">
          <property name="title">
           <string>绑定本地地址</string>
          </property>
          <layout class="QVBoxLayout" stretch="1,1,1,1,1">
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_tcpServer_localIP">
             <property name="text">
              <string>IP地址：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_tcpServer_localIP"/>
           </item>
           <item>
            <widget class="QLabel" name="label_tcpServer_localPort">
             <property name="text">
              <string>端口号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_tcpServer_localPort">
             <property name="inputMask">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_tcpServer_bind">
             <property name="text">
              <string>创建</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" rowspan="3">
         <widget class="QGroupBox" name="groupBox_tcpServer_recv">
          <property name="title">
           <string>网络数据接收</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTextEdit" name="textEdit_tcpServer_recv"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QGroupBox" name="groupBox_tcpServer_Send">
          <property name="title">
           <string>网络数据发送</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,1,1,0">
           <item row="1" column="0">
            <widget class="QLabel" name="label_tcpServer_clientList">
             <property name="text">
              <string>客户端列表:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_tcpServer_clientList"/>
           </item>
           <item row="0" column="0" colspan="4">
            <widget class="QTextEdit" name="textEdit_tcpServer_send"/>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="pushButton_tcpServer_send">
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_tcpClient">
       <attribute name="title">
        <string>TCP Client</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_bind_3">
          <property name="title">
           <string>连接服务器地址</string>
          </property>
          <layout class="QGridLayout" name="_2" rowstretch="0,0,0,0,0" columnstretch="0" rowminimumheight="0,0,0,0,0" columnminimumwidth="0">
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <property name="leftMargin">
            <number>12</number>
           </property>
           <property name="rightMargin">
            <number>12</number>
           </property>
           <property name="bottomMargin">
            <number>12</number>
           </property>
           <property name="horizontalSpacing">
            <number>-1</number>
           </property>
           <item row="4" column="0">
            <widget class="QPushButton" name="pushButton_tcpClient_Connect">
             <property name="text">
              <string>连接</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="lineEdit_tcpClient_serverPort">
             <property name="inputMask">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="comboBox_tcpClient_serverIP"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_tcpServer_localPort_3">
             <property name="text">
              <string>端口号：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_tcpServer_localIP_3">
             <property name="text">
              <string>IP地址：</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_tcpServer_Send_2">
          <property name="title">
           <string>网络数据发送</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8" columnstretch="0,0,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item row="0" column="0" colspan="3">
            <widget class="QTextEdit" name="textEdit_tcpClient_send"/>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_tcpClient_send">
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_tcpServer_recv_2">
          <property name="title">
           <string>网络数据接收</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item row="0" column="0" rowspan="2" colspan="2">
            <widget class="QTextEdit" name="textEdit_tcpClient_recv"/>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_tcpClient_recv">
             <property name="text">
              <string>接收</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_udp">
       <attribute name="title">
        <string>UDP</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBox_tcpServer_localIP</tabstop>
  <tabstop>lineEdit_tcpServer_localPort</tabstop>
  <tabstop>pushButton_tcpServer_bind</tabstop>
  <tabstop>textEdit_tcpServer_recv</tabstop>
  <tabstop>comboBox_tcpServer_clientList</tabstop>
  <tabstop>textEdit_tcpServer_send</tabstop>
  <tabstop>pushButton_tcpServer_send</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
